reference/runtime-environment.html.md

```diff
@@ -32,6 +32,21 @@ Not to be confused with the [HTTP header](/docs/reference/runtime-environment/#f
 ### `FLY_IMAGE_REF`
 **Docker Image Reference**: The name of the Docker image running this container. Useful if your application needs to launch Machine instances of itself to scale up-or-down. Read about how [a Rails app runs Machines for scale-to-zero background workers](https://fly.io/ruby-dispatch/rails-background-jobs-with-fly-machines/). `registry.fly.io/my-app-name:deployment-01H9RK9EYO9PGNBYAKGXSHV0PH` is an example of the Docker Image Reference's format.
 
+### `FLY_MACHINE_ID`
+**Machine ID**: The unique ID that flyctl commands and the Machines API use to identify a Machine. You'll also see it in the [logs](/docs/flyctl/logs/).
+
+### `FLY_MACHINE_VERSION`
+**Machine Configuration Version**: Every Machine configuration has an associated version. Whenever you update a Machine's configuration (including when you update its Docker image), it gets a new `FLY_MACHINE_VERSION`. The exception to this is that changing only the Machine's metadata doesn't trigger a new version.
```

Comment:
How about this? It's a bit lifeless, but the info is there.

```suggestion
**Machine Configuration Version**: Any update to a Machine's configuration, including its Docker image, generates a new `FLY_MACHINE_VERSION`. Updating metadata alone does not trigger a new version. `FLY_MACHINE_VERSION` has the same value as `Instance ID` in the output of `fly machine status`. 
```

The original wording echoed the structure of similar entries in the existing doc. This one needs a certain amount of explanation at this time, because it's not intuitive and as far as I'm aware it's not mentioned elsewhere. That can be corrected (and the doc as a whole could be tidied) at some time past the scope of this PR.

Reply:
No, this is great! I love how it's more descriptive. This was probably the hardest one for me to come up with information to talk/describe about.